### ISSUE TITLE
signin in local dev environment doesn't work

### ISSUE THREAD

**Issue by willkg** [mozilla/standup#461]
When trying to sign in in the local dev environment, I get an error from the Mozilla SSO saying our callback is not allowed.

The problem is that the django app now runs as `app:8000` and before it ran as `localhost:8000`. I think we just need to add that to the Mozilla SSO thing.

I did a request to update our client account thing.

**Comment by willkg** [comment]
So, what happens is that mozilla-django-oidc generates the callback url using the host and port of the django server. So right now it's thinking "app:8000". If we move it around, then it'll still be wrong.

I looked at whether we could tell mozilla-django-oidc to not do what it's doing and I don't think we can. Pretty sure the problem is here:

https://github.com/mozilla/mozilla-django-oidc/blob/6f84a68147b4a41b686c550c4d05533be993303c/mozilla_django_oidc/auth.py#L227

I think we need to do something different. Instead of having browsersync proxy, we have two separate web services one serving assets and the other serving django. So Django app would be `http://localhost:8000/` and the assets server would be `http://localhost:3000/` and we set STATIC_URL to something like `http://localhost:3000/`.

Would that work?
